extensions/integration-tests/src/test/dacpac.test.ts

```diff
@@ -25,7 +25,7 @@ suite('Dacpac integration test suite @DacFx@', () => {
 		console.log(`Start dacpac tests`);
 	});
 
-	test('Deploy and extract dacpac @UNSTABLE@', async function () {
+	test('Deploy and extract dacpac', async function () {
 		this.timeout(5 * 60 * 1000);
```

Comment:
the tests are passing in DacFx Integration Tests pipeline. The only issue I've been seeing is the timeout issue

Reply:
Are you saying these ones I've marked stable have been failing? Should I leave them as unstable for the time being then?

Reply:
they are not failing in the pipeline I added but I had to set the timeout way higher. I'm saying they might still fail because of timeout unless you change the timeout to 10/20 minute maybe

Reply:
I'm not going to make any test changes with this PR - I'll leave that up to the owners 😄 

It sounds like I should keep this as unstable until you're sure increasing the timeout helps. That sound reasonable?